refinery/data_set_manager/utils.py

```diff
@@ -464,11 +464,10 @@ def update_annotated_nodes(
         # TODO: Not happy about this hack at all.
         error_message = (
             "Exponential explosion! Creation of {} annotated nodes for {} "
-            "nodes of type {} is stopped!"
+            "nodes of type {}"
         ).format(total_attrs, num_nodes_of_type, node_type)
 
         logger.error(error_message)
```

Comment:
Should this be `logger.warning(warning_message)` since import is no longer prevented?

Reply:
Would make sense to leave it as `logger.error` since this state, as far as we know, is an error? 

Either way is fine with me.

Reply:
Well, it turned out that it is definitely not an error for some data sets.

Reply:
Even though these Datasets can now import without this check in place there is still the deeper issue of many `Attributes` being created with duplicated info from parsing their ISATabs and this logic still catches that. Maybe leaving it as an error message would keep us on our toes when seeing this in the future.